packages/opentelemetry-node/README.md

```diff
@@ -1,4 +1,4 @@
-# OpenTelemetry Node SDK
```

Comment:
Should we still document the fact that this is an SDK at least in the READMEs?

Reply:
It's in the base modules. This package first of all automatic instrumentation of Node.js on top of the SDK.

Reply:
We can still change some wording in the docs as we go. For now, it's important that the package names are right when we publish.